tests/integration_test/services_test.py

```
@@ -6555,6 +6573,325 @@
     CognitionData.objects(bot=pytest.bot).delete()
     CognitionSchema.objects(bot=pytest.bot).delete()
 
+@pytest.mark.asyncio
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)